packages/bpk-foundations-web/tokens/base.default.scss

```diff
@@ -502,29 +502,29 @@ $bpk-grid-container-padding: 1.5rem !default;
 /// @group grids
 $bpk-grid-gutter: 1.5rem !default;
 /// @group font-sizes
-$bpk-h-1-font-size: 2.625rem !default;
+$bpk-h-1-font-size: 3rem !default;
```

Comment:
While we won't be using these tokens anymore in mixins, I'm wondering if it's worth it for those 3 use cases to have this token use xxxl under the hood as the heading-1 style does 🤔

Reply:
Yeah sure can do, these only changed because they were using the font tokens under the hood, but I guess if they are going away and we don't want people to use them then we should keep them to this in order to force change?